docs/source/llms/transformers/tutorials/fine-tuning/transformers-peft.ipynb

```diff
@@ -0,0 +1,1281 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Fine-Tuning Open-Source LLM using QLoRA with MLflow and PEFT"
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "metadata": {},
+   "source": [
+    "<p></p>\n",
```

Comment:
Found that we can insert the download link in the notebook using "raw" cell, so we can remove the [workaround](http://localhost:8000/llms/transformers/index.html#download-the-use-case-tutorial-notebooks-to-try-them-locally) of putting the button in index page. If this sounds good, I will apply same to the other Transformers notebook in the follow-up cc: @BenWilson2

Reply:
No, I agree with you. Let's file a ticket for next sprint to migrate to the raw embed within notebook approach. It's cleaner. We should do this for all notebooks in all sections. Probably a 0.5 point ticket :)